front/src/config/demo.json

```diff
@@ -169,7 +169,7 @@
     "humidity": 0.99,
     "pressure": 1005.09,
     "datetime": "2019-05-09T04:27:57.000Z",
-    "units": "si",
```

Comment:
Be careful, there are still mention of "si" in the code (search in the code to find where)

Reply:
? weird...in my branch there is no more mentions on "si"...even in `demo.json` file it's "metric"

Reply:
<img width="1552" alt="Screenshot 2020-06-01 at 11 13 35" src="https://user-images.githubusercontent.com/7365207/83394606-2881c980-a3f9-11ea-9c13-c870e01f4b4d.png">
<img width="1552" alt="Screenshot 2020-06-01 at 11 13 46" src="https://user-images.githubusercontent.com/7365207/83394621-2cade700-a3f9-11ea-9803-8388ccb444d3.png">